src/slurm_plugin/cluster_event_publisher.py

```diff
@@ -597,6 +642,21 @@ def _failed_node_supplier(unhealthy_static_nodes: List[SlurmNode], failed_nodes:
                         }
                     }
 
+    def _nodemapping_supplier(self, nodes: List[SlurmNode]) -> Iterator:
+        for node in nodes:
+            yield {
+                "detail": {
+                    "nodes": [
```

Comment:
With this we are creating one detail for each node and only have a single node in the `nodes` list , this is not what we expect to have.
Could you fix this by including all the nodes into the `nodes` list?

Reply:
After changing this, could you add an example test output of the json event in the PR description?